tests/assets/default.conf

```diff
@@ -30,4 +31,7 @@ enable-protected-configs yes
 enable-debug-command yes
 enable-module-command yes
 
-propagation-error-behavior panic
\ No newline at end of file
+propagation-error-behavior panic
+
+# Make sure shutdown doesn't fail if there's an initial AOFRW
+shutdown-on-sigterm force
```

Comment:
so it looks like there's no test that checks that the default is to refuse shutdown in there's an initial AOFRW?
let's add one.